carball/analysis/stats/utils/pandas_utils.py

```diff
@@ -33,4 +33,4 @@ def sum_deltas_by_truthy_data(data_frame: pd.DataFrame, truthy_frames: pd.Series
     """
     truthy_frames = truthy_frames.rename('truthy')
     combined_data = pd.concat([truthy_frames, data_frame['game', 'delta'].rename('delta')], axis=1)
-    return combined_data.loc[combined_data['truthy'] == True].sum().rename(columns={'delta': 'true_values'}).delta
+    return combined_data.loc[combined_data['truthy'] == True].sum().loc['delta']
```

Comment:
Why this change?

Reply:
to fix the 50+ tests that were broken. there we two issues with this line as i was debugging:
1. sum() returns a dataframe that didn't need to be renamed to get the delta
2. accessing the delta value by the `delta` index didn't seem to work the same way.

i am mostly unfamiliar with a lot of this stuff, so maybe there was a reason it was the way it was, but the API seems to have changed and i guess this is the "new" way to do it "right"?

Reply:
okay that makes sense.   But do you know what version it changed?

Because we are setting a new "minimum" pandas if this new way does breaks the old "minimum" pandas then we need to enforce the higher version

Reply:
i don't think it does, otherwise we would have seen the tests break right? with my admittedly SUPER limited knowledge and exposure, my guess is they had a more flexible set of tools or maybe they had a less "intuitive" api where you had to rename an index to do an attribute access a la `df.thingToAccess`. maybe they also introduced certain restrictions around these things? here's the latest documentation on accessing an index by attribute name: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#attribute-access

Reply:
no that would not be the case because they all are installing pandas 1.03

Remember that the tests use a "fresh" install not an existing install.  So they do not run at the minimum.

Reply:
We should likely just change the minimum to the current version that is being run.  1.0.3